ferrocene/ci/scripts/calculate_parameters.py

```diff
@@ -44,6 +44,7 @@
 
 GENERIC_BUILD_STD_TARGETS = [
     "aarch64-unknown-none",
+    "thumbv6m-none-eabi",
     "thumbv7em-none-eabi",
```

Comment:
Maybe we could sort this array? Just so it's obvious where new targets need to go.